server/middleware/auth.js

```diff
@@ -38,13 +87,15 @@ const adminOnly = (req, res, next) => {
 
 const userOnly = (req, res, next) => {
   const { userId } = req.user;
-  if (!userId) {
-    return res.status(403).send({
-      status: 403,
-      error: 'Unauthorized Access. For authenticated users only',
-    });
-  }
+  // Fix this: doing nothing!
```

Comment:
@Bastieno can you please attend to this redundancy in the code base?

Reply:
Done that.